### PR TITLE
Updated Kinematics schema and init files

### PR DESCRIPTION
**Pull request by manuparra**

### CODE REVIEW

**Comment by manuparra** [comment]
@axshen Check it and merge it if your are agree with the changes. Thanks Austin.

File: src/03-kinematics.sql [hunk 1]
```diff
@@ -0,0 +1,26 @@
+\connect wallabydb
+
+CREATE TABLE wallaby.kinematic_model (
+  "source_name" varchar PRIMARY KEY,
```
Comment (axshen) [inline src/03-kinematics.sql:4]: This table should also have a id field for its primary key right? 
`"id" BIGSERIAL PRIMARY KEY`

Comment (manuparra) [reply src/03-kinematics.sql:4]: I thinks so. We can wait @venkat to confirm it.

Comment (manuparra) [reply src/03-kinematics.sql]: Solved in the latest commits. Thanks !

**Pull request merged**